headers.go

```diff
@@ -75,6 +75,12 @@ func (h *EthernetHeader) WithVLANID(vid uint16) *EthernetHeader {
 	return h
 }
 
+// WithEthernetType sets string type Ethernet-Type of the Ethernet header to the specified value
+func (h *EthernetHeader) WithEthernetType(ethernetType string) *EthernetHeader {
```

Comment:
to be consistent with the spec, let's call this property "etherType" throughout and make it an "int" type

Reply:
thanks for checking. I made the changes as suggested.